app/Http/Middleware/LangMiddleware.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+
+class LangMiddleware
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+        if ($lang = $request->session()->get('lang')) {
+            \App::setlocale($lang);
```

Comment:
```suggestion
            App::setlocale($lang);
```
Anh `use App;` ở trên thay vì dùng \App nhé.